test/tests.bats

```diff
@@ -678,7 +678,7 @@ EOF
     -v "$(duplicate_config_for_container without-accounts/ without-accounts-deleting-user)":/tmp/docker-mailserver/ \
     "${IMAGE_NAME:?}" /bin/sh -c 'delmailuser -y user3@domain.tld'
   assert_success
-  [ -z "${output}" ]
+  [[ -n ${output} ]]
```

Comment:
Was this a bugfix or is the `-n` a typo/mistake?

```suggestion
  [[ -z ${output} ]]
```

Reply:
That is a typo, thank's for spotting!